### ISSUE TITLE
Balance missing as well. 

### ISSUE THREAD

**Issue by visionguild** [iotaledger/wallet#382]
using 2.5.1 and it shows my transferring in 120 iota and then has it sent out to another address.  I never sent any iota.  I didn't transfer this from Bitfinix.  Did someone hack my wallet.  How do I get my balance back.  Slack is full.  I think maybe Iota isn't a company worth supporting.  Was about to move more Iota into my wallet.  Think I'll just sell it instead.

**Comment by ilore** [comment]
I have exactly the same problem. What is happening?
I only want to have access to my balance in order to send it to any other wallet, because this wallet is a completely mess ...

**Comment by arichoogs** [comment]
same problem, i read about foundation addresses. not sure if this what is happening.

**Comment by paulhandy** [comment]
This _just_ happened? Can you provide more information? 

For example, could you describe the method that you used to generate your seed?

**Comment by visionguild** [comment]
Issue report seems suspect!  What?  Yeah, I have nothing better to do than sit around trying to make issues with Iota.  I downloaded the wallet.  Went through the process it let me through to generate the seed.  Saved that seed where I could get it easy and saw that yes, my balance did come in, then it went out.  Bunch of transactions.  All to the same address, some pending, others confirmed.  Confirmed withdrawals of all my Iota.  Balance 0.  I already sold all the rest of my Iota.  I doubt I'll ever get this back.  Certainly wouldn't ever put anything back in it.  Saw that a bunch of peeps have been having this problem with this version.  Guess it's our loss.

**Comment by ilore** [comment]
I downloaded the wallet from the official site, and created my account following the instructions in the wallet: help / FAQ / help / Creating a new account.

In my transfers history there is an incoming transfer between bitfinex and my wallet that I did when I purchased MIOTAs. But, the strange thing is that a few hours later there is an outcoming transfer from my wallet to an unknown adress. I didn´t do that outcoming transfer.

**Comment by arichoogs** [comment]
I also used the iotas recommended website for seed creation, which was part of the set up guide for installing the wallet.

**Comment by arichoogs** [comment]
Maybe they will release a new snapshot of the foundation addresses used from this week.

**Comment by visionguild** [comment]
So I guess if Iota doesn't want to help you they call your issue suspect and close the discussion. That's what they did on my other thread I started on this issue.  I think we're all screwed.  Haven't seen anywhere that they've helped.

**Comment by th0br0** [comment]
As Paul said: please provide proof.

Valid forms of proof would be: addresses belonging to your seed from which you claim funds were moved or transaction hashes of the transactions moving the funds.

**Comment by ilore** [comment]
In my case:

![iotatransfers](https://user-images.githubusercontent.com/32924279/31831974-98418e7e-b5c5-11e7-85c6-73bf7c6868f0.jpg)

The hash from the incoming transfer (12/10/2017 10:48) is this: UKUQILMPSHNFMNYHFQX9DXWQYTBALJNOXTOIGJFXOM9LNVUTAMFF9NWTYTOMEHNPMKCXNPYPLBKQ99999

The hash from the first outcoming transfer (12/10/2017 14:22) is this: TYEHHXIHDGRYNRDLQPHJTVTSEHONJGZEDXL9V9JHXYPCQEIIQXEXIPIKWWKIVWWOVEDMTWVVPZSWZ9999

The hash from the second outcoming transfer (12/10/2017 14:19) is this: BEJPSXQPLPJS9ZFOC9YU9QVYHKJYGPYAJCNGYSDBHELWLGABCFCFKBCBXRQUPOUUDGNYABEICQUA99999

The hash from the third outcoming transfer (12/10/2017 14:17) is this: TQZUFGPKONCBZLOOUGMF9FEECVHQBYRTXCZWRBVBJBEIVD9SFTCCLVHPIPHXNQPVFUAHMNWUWRNWZ9999

**Comment by th0br0** [comment]
@ilore in your case it appears to be a seed compromise. How did you generate your seed?

**Comment by ilore** [comment]
@th0br0 I followed the instructions in the wallet: help / FAQ / help / Creating a new account. 

I wrote a long password (my seed), and I didn't share it with anybody, I am completely sure.

**Comment by arichoogs** [comment]
![121](https://user-images.githubusercontent.com/32939451/31833119-8f511786-b597-11e7-836b-9ac406161462.JPG)

Address that it was sent to.
ACTQYNEDMF9JLWYRIXFED9AFOUHSTUIXOQYJVBGE999QKLGGFHZWSUOZBVIZBVOAJSGRKYJMPPMPJXOPAHVX9UFEXX

**Comment by arichoogs** [comment]
got my seed from the same place ilore did.

**Comment by arichoogs** [comment]
THIS WEBSITE : https://ipfs.io/ipfs/QmdqTgEdyKVQAVnfT5iV4ULzTbkV4hhkDkMqGBuot8egfA

**Comment by visionguild** [comment]
![screen shot 2017-10-20 at 10 40 00 am](https://user-images.githubusercontent.com/32921004/31834577-46e464d0-b583-11e7-9290-bc63ea31cd62.png)
![screen shot 2017-10-20 at 10 40 36 am](https://user-images.githubusercontent.com/32921004/31834581-48ee7bbc-b583-11e7-97fd-af2a70b24d01.png)

I'm trying to get it to show me the bundle so I can copy the address it went to but it's not letting me now.  Show Bundle link does nothing.

**Comment by visionguild** [comment]
I think this is the one it went too.  finally showed me the bundle.  I'm new at this so I don't know how to check things.  

OOKXLSCJCKNQWZIMFCAIVZHVCWRFEMMGAIXHXKEKHJHXOBRTDZFRVGHMOZDDOYVGTVWCLPVAVASXKUZIW9NFS9TXVW

![screen shot 2017-10-20 at 10 45 34 am](https://user-images.githubusercontent.com/32921004/31834729-d5581f90-b583-11e7-8037-fd8f0f4fc149.png)

**Comment by visionguild** [comment]
screen shots are on their.  Tell me if you need something else.

**Comment by arichoogs** [comment]
If this was one person taking our money would it be the same wallet address the money is going to??

**Comment by wonkytonky** [comment]
no he can create as many new address as he wants  .. like you can too

**Comment by visionguild** [comment]
i wonder how many of us have to lose our money before they take it
seriously.  I sold all my Iota yesterday.  Don't want to support an
organization that is that careless.  Be great to get this back but it's
only 55 dollars so I can take the loss.  Word will get around about this.
 keep posting on Reddit I guess as there is no Iota that we can petition.

**Comment by paulhandy** [comment]
I assure you that we take this quite seriously; but investigation of these issues takes time and requires information like what you just provided.

**Comment by arichoogs** [comment]
They directed us to a site for seed generation that was compromised. How can you not blame them a bit.  There has to be some sort of reliability/trust within the system of this currency will never work.

**Comment by visionguild** [comment]
I have to agree with arichoogs.  I've been trading cryptos for over a year
and never have I had a wallet compromised like this.  Iota sent us to this
address and many of us lost our funds.  You can just wipe your hands of it
if you want, but I think in the end this kind of thing only hurts our
community.  I'm going to make sure that I post this on Reddit as much as
possible to let people know that Iota is not as secure as it should be and
I'll list all the people I've seen with this problem and how the devs have
no real response other than to say, it was maybe our fault and you're out
of luck. I've already told all my crypto friends.  Two have divested of
Iota because they don't know where to keep it now.  I have at least 10
wallets I work with and Iota is the only one that has lost my money in over
a year.  Only Iota.  You guys should be proud.  So glad it was only 55
bucks.  If it was more, I'd probably start a campaign against you guys
information wise.  People need to know this is happening with your wallet.

**Comment by oragnejang** [comment]
I have also same problem.
We just generate the seed your guide. It is terroble.
I lost 500iota. 
Do not use IOTA. YOU MUST LOST YOUR MONEY JUST USING THE WALLET.

**Comment by arichoogs** [comment]
They devs clearly dont care because we lost our coins weeks ago and they still havent fixed the problem.
I wonder who is taking out money?

**Comment by arichoogs** [comment]
www.scam.com download my new wallet

**Comment by oragnejang** [comment]
![31664382-253cdd26-b380-11e7-8c92-2c2e95ac704b](https://user-images.githubusercontent.com/11746626/32635573-38614208-c5f4-11e7-9648-2a21a7399e4d.png)
This is my status.
It is exactly same.
2 or 3times out from my wallet to unkown address. I think It just wallet bug.

**Comment by georgpukk** [comment]
Welcome me to the club.
I'm going to post my information here just for the record.

I have some similarities with ilore and visionguild.
I withdraw from bitfinex. And right after that I get hacked. I was using v. 2.5.4.
https://iotasear.ch/address/AXVSMGCCTDXOOOQUGLPSHCSTKZCHAHRCTKIYQRDOZXWQLZAIF9FJCIBOGHF9RANIPMFIURQBAFPVNZLJYFB9SYKRZ9

I generated my seed in Keepass. Kept it encrypted with Keepass on a flash drive.
Only copied the seed between the flash drive and Light Wallet with Two-Channel Auto-Type Obfuscation enabled.

**Comment by visionguild** [comment]
Sorry to hear this sir.  The shitty thing is the devs don't give a shit
about us.  I got labeled as spreading FUD and dismissed by some dick dev
and then after others came out and said the same thing they actually
responded only to say, "hmm, oh well".  I wonder where people keep their
Iota?  I'm keeping what little I have left on Binance as I don't have any
other place to put it.  Sorry you got fucked by Iota's shitty wallet and
non support.  Kinda wish their price wasn't increasing.  I don't think they
deserve it.

**Comment by visionguild** [comment]
"I assure you we take it quite seriously"  you said a month ago.  Now I
just got another email from another person fucked by this shitty wallet.
 never saw mine again and never heard from you caring devs that take all
this so seriously.  Question remains, how many people are gonna lose money
before you wake up.  Looks like the total is up to about 7 on this post.
Way to go!!  I feel so supported.

**Comment by arichoogs** [comment]
lol they release new wallet versions and dont correct the problem...what assholes .... iota is trash

**Comment by supersicco** [comment]
Hi, I've had the exact same thing happen to me. It shows my iota being sent somewhere else and confirmed. Does this really meant the iota is gone and no way of getting it back? What can i do? Please help..
![iota](https://user-images.githubusercontent.com/33969395/33225250-fe9c128e-d1c7-11e7-95c9-234015a7692a.jpg)

**Comment by supersicco** [comment]
so are we saying that if we followed the "iota guide" from 
https://iota.guide/seed/how-to-generate-iota-wallet-seed/
that means our seed has been compromised?

**Comment by edwinckc** [comment]
Same, followed the guide to generate a seed. First couple attempts to withdraw from Bitfinex didn't work even after 12+ hours. Left the last attempt for about a day, just checked and everything's gone. Sigh...

![image](https://user-images.githubusercontent.com/2934985/33225594-ba4939e4-d137-11e7-8869-df689d8d46ca.png)

**Comment by arichoogs** [comment]
yep its gone... to bad we cant warn people before ...or iota devs could simply change the seed website but that wold be to simple and then they wouldnt be able to rob you

**Comment by arichoogs** [comment]
bitfinex is the problem ive come to the conclusion. look up what is happening with tether. these guys r crooks

**Comment by arichoogs** [comment]
iota still has a shit wallet but either way

**Comment by edwinckc** [comment]
I agree Bitfinex has been shady with the whole Tether fiasco, but they only had access to the wallet address, not the seed. Theoretically they shouldn't be able to send money away from your own wallet. If the funds were sent from the IOTA account on Bitfinex then that'd be a different issue I think.

**Comment by visionguild** [comment]
I agree man!  Don't you wish you could find them and take their money.
They are so careless with ours.

**Comment by wonkytonky** [comment]
never use online seed generator, EVER. there a lot of evil persons out there creating guids and help and online seedgent purely to steal your money. 

it's better to make a seed by yourself. rolling a alphabet dice, or using a good password randomizer like https://keepass.info/, or /dev/urandom 





Van: "supersicco" <notifications@github.com> 
Aan: "iotaledger/wallet" <wallet@noreply.github.com> 
Cc: "bjorn" <bjorn.c@telenet.be>, "Comment" <comment@noreply.github.com> 
Verzonden: Zaterdag 25 november 2017 00:48:00 
Onderwerp: Re: [iotaledger/wallet] Balance missing as well. (#382) 



so are we saying that if we followed the "iota guide" from 
[ https://iota.guide/seed/how-to-generate-iota-wallet-seed/ | https://iota.guide/seed/how-to-generate-iota-wallet-seed/ ] 
that means our seed has been compromised?
